Xamarin.Essentials/Contacts/Contacts.android.cs

```diff
@@ -22,6 +22,29 @@ static async Task<Contact> PlatformPickContactAsync()
             return null;
         }
 
+        static async Task<IEnumerable<Contact>> PlatformGetAllAsync()
+        {
+            await Task.CompletedTask;
+            return GetAll();
```

Comment:
You can also do `Task.FromResult(...)` I think. Then no need for the async/await part at all.

Reply:
this is a draft

Reply:
Oh yeah. Just throwing comments out for discussion here. Thanks for getting started on this.